[docs] EAS Build with submodules (#13595)
expo/expo: docs/pages/build-reference/how-tos.md

  }
}
```

## How to use git submodules

> Create a [secret](/build-reference/variables/#using-secrets-in-environment-variables) with base64 encoded private ssh key that have permission to access submodules repositories and add `eas-build-pre-install` npm hook that is checking out those submodules e.g.

Reviewer: ```suggestion
Create a [secret](/build-reference/variables/#using-secrets-in-environment-variables) with a base64 encoded private ssh key that has permission to access submodules repositories and add the `eas-build-pre-install` npm hook that is checking out those submodules, e.g.
```

> Create a [secret](/build-reference/variables/#using-secrets-in-environment-variables) with a base64 encoded private ssh key that has permission to access submodules repositories and add the `eas-build-pre-install` npm hook that is checking out those submodules, e.g.

Reviewer 2: ```suggestion
First, create a [secret](/build-reference/variables/#using-secrets-in-environment-variables) with a base64 encoded private SSH key that has permission to access submodules repositories. Next, add an `eas-build-pre-install` npm hook to check out those submodules, for example:
```

```bash
#!/usr/bin/env bash

mkdir -p ~/.ssh

> # uncomment line bellow and specify url for remote "origin" if your submodules

Reviewer: ```suggestion
# uncomment the line below and replace the origin's URL if your submodules
```

> # are defined with relative urls in .gitmodules

Reviewer 2: this could use more context
